tailwind.config.js

```diff
@@ -102,6 +119,69 @@ const config = {
         md: '0px 1px 2px 0px #000000 6%, box-shadow: 0px 1px 3px 0px #000000 10%',
         lg: '0px 4px 6px -2px #000000 5%, 0px 10px 15px -3px #000000 10%',
       },
+
+      // unfortunately @tailwindcss/typeography doesn't ship types
```

Comment:
Can you move this to a PR for `saga`? Because everyone will benefit of that.

Reply:
Done: https://github.com/distributeaid/distributeaid.org/pull/822